### ISSUE TITLE
Can I use lr_scheduler of Pytorch in model.fit ?

### ISSUE THREAD

**Issue by kyuchoi** [havakv/torchtuples#24]
First of all, thank you for your great works !!

Can I use lr_scheduler of Pytorch in model.fit ?
I've noticed that there is lr_scheduler.py in this repo, however, I couldn't find how to use it in examples.
If it's possible, then how can I modify the following code to use the scheduler for learning rate?

log = model.fit(x_train, y_train, batch_size, epochs, callbacks, val_data=val)

Many thanks

**Comment by havakv** [comment]
This question is currently discussed in the pycox repo https://github.com/havakv/pycox/issues/103
